src/voxel.py

```diff
@@ -105,6 +105,8 @@ def __init__(self):
     def _initialise_data(self):
         # Our scene data
         self._data = self.blank_data()
+        # Create empty selection
+        self._selection = []
```

Comment:
It seems sets don't have append either. I'm keeping it a list for now. Shouldn't be a problem.